sync-core/src/main/java/com/cloudant/http/HttpConnection.java

```diff
@@ -79,6 +84,16 @@
 
     private static String userAgent = getUserAgent();
 
+    public final List<HttpConnectionRequestFilter> requestFilters;
+    public final List<HttpConnectionResponseFilter> responseFilters;
+
+    //we need to store the response code separately so we don't send a request
+    //before we are ready
+    private int responseCode = -1;
```

Comment:
CookieFilter was using it originally when we had a single interface method for filtering requests and responses, will remove